Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.py

```diff
@@ -263,6 +263,9 @@ def get_endpoints(self,
                 json_data={}
             )
             endpoints = reply.get('reply')[search_from:search_to]
+            for endpoint in endpoints:
+                if not endpoint.get('endpoint_id'):
+                    endpoint['endpoint_id'] = endpoint.get('agent_id')
```

Comment:
how do we know `agent_id` and `endpoint_id` are the same ?

Reply:
![Playground 2020-06-04 10-42-20](https://user-images.githubusercontent.com/44666568/83729501-6167ab80-a650-11ea-875c-ff0b91d812cb.png)
![Playground 2020-06-04 10-43-19](https://user-images.githubusercontent.com/44666568/83729507-64629c00-a650-11ea-997c-7a27ebaec6d4.png)
 **It is the same command - was returning once `agent_id` once `endpoint_id`**